src/solvers/refinement/string_refinement.cpp

```diff
@@ -510,8 +510,16 @@ void string_refinementt::set_to(const exprt &expr, bool value)
 
     // Preprocessing to remove function applications.
     const exprt &lhs=eq_expr.lhs();
-    debug() << "(sr::set_to) " << from_expr(ns, "", lhs)
-            << " = " << from_expr(ns, "", eq_expr.rhs()) << eom;
+
+    try
+    {
+      debug() << "(sr::set_to) " << from_expr(lhs)
+              << " = " << from_expr(eq_expr.rhs()) << eom;
+    }
+    catch (std::string s)
```

Comment:
I think this was happening when the namespace was not given but another commit added this, so this may not be necessary anymore. I will remove this part of the commit and test it.